modules/jsonld/src/main/scala/ch/epfl/bluebrain/nexus/rdf/jsonld/JsonLd.scala

```diff
@@ -0,0 +1,147 @@
+package ch.epfl.bluebrain.nexus.rdf.jsonld
+
+import cats.Monad
+import cats.data.EitherT
+import ch.epfl.bluebrain.nexus.rdf.Iri
+import ch.epfl.bluebrain.nexus.rdf.Iri.AbsoluteIri
+import io.circe.{Json, JsonObject}
+import io.circe.syntax._
+import cats.implicits._
+
+object JsonLd {
+
+  /**
+    * Resolve all IRIs inside @context.
+    *
+    * @param json     [[Json]] with context to resolve
+    * @param resolver context resolver
+    * @return [[Json]] with all the context IRIs resolved
+    */
+  def resolveContext[F[_]: Monad](
```

Comment:
maybe we should avoid explicitly returning an `EitherT`... It seems an unnecessary burden when not used with `EitherT` and fairly simple to lift an `F[Either[A, B]]` to an `EitherT[F, A, B]`; thoughts?